editor/components/arrangement-toolbar/index.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
I will put on my todo-list to create a labelled blueprint for the UI as a start, that can hopefully grease the discussion for what the individual components and classes should be called.